src/components/MatchScore/MatchScore.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+import styles from './matchScore.module.scss';
+import { classNames } from '../../shared/utilities';
+import { FillType, MatchScoreProps } from './MatchScore.types';
+
+export const MatchScore: FC<MatchScoreProps> = ({
+    className,
+    score = 0,
+    total = 5,
+    hideLabel = false,
+}) => {
+    const absTotal = Math.abs(total);
+    const matchScoreClasses = classNames(className, styles.matchScoreContainer);
+    return (
+        <div
+            className={matchScoreClasses}
+            role="progressbar"
```

Comment:
The role `progressbar` will get flagged by A11y test. Will need to use an ariaLabel prop and concatenate the values as needed.